overlord/snapstate/snapstate.go

```diff
@@ -86,6 +86,8 @@ const (
 	snapdDesktopIntegrationSnapID = "IrwRHakqtzhFRHJOOPxKVPU0Kk7Erhcu"
 )
 
+const defaultMaxInhibitionDays = 14
```

Comment:
tiny nitpick, I think it would be a good idea to have `defaultMaxInhibitionDays` closer to `maxInhibitionDuration` since it is the only place using it.

Reply:
Sorry, forgot to do this. Done and rebased to avoid conflict.